ecosystem/typescript/sdk/src/tests/e2e/aptos_client.test.ts

```diff
@@ -16,18 +16,12 @@ import { bcsSerializeUint64, bcsToBytes } from "../../bcs";
 import { AccountAddress, Ed25519PublicKey, stringStructTag, TypeTagStruct } from "../../aptos_types";
 import { Provider } from "../../providers";
 import { BCS } from "../..";
-import { VERSION } from "../../version";
 
 const account = "0x1::account::Account";
 
 const aptosCoin = "0x1::coin::CoinStore<0x1::aptos_coin::AptosCoin>";
 
-test("call should include x-aptos-client header", async () => {
-  const client = new AptosClient(NODE_URL, { HEADERS: { my: "header" } });
-  const heders = client.client.request.config.HEADERS;
-  expect(heders).toHaveProperty("x-aptos-client", `aptos-ts-sdk/${VERSION}`);
-  expect(heders).toHaveProperty("my", "header");
-});
```

Comment:
No longer testing for the header?

Reply:
yep, testing in a different place now https://github.com/aptos-labs/aptos-core/pull/8738/files#diff-9187cd50043dfc9d3829abdb7a919d2497b633279daa100db09194b2d6509366R5